wallet/src/ui/app/hooks/useFormatDate.ts

```diff
@@ -0,0 +1,12 @@
+// Copyright (c) 2022, Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+// TODO - handle multiple date formats
+// Wed Aug 05
+export default function useFormatDate(date: Date) {
```

Comment:
This likely doesn't need to be a hook, I can't really imagine anything that'd need to be done here that would tie into react?